LICENSE

```diff
@@ -0,0 +1,340 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/license/agpl-3.0.txt should be in